menu/menu_displaylist.c

```diff
@@ -1696,6 +1696,7 @@ static unsigned menu_displaylist_parse_system_info(file_list_t *list)
             cpu_arch_str, sizeof(cpu_arch_str));
 
       strlcpy(cpu_str, cpu_text_str, sizeof(cpu_str));
+      strlcat(cpu_str, ": ", sizeof(cpu_str));
       strlcat(cpu_str, " ", sizeof(cpu_str));
```

Comment:
This line needs to be removed.

Reply:
Line 1700 with the blank " "?

Reply:
Yes. Space is already added by the new line. If you don't remove that line, you've a colon followed by two whitespaces.

Reply:
Grand, will do. Cheers!

Reply:
Line has been removed